infra/eks/maskrcnn/templates/maskrcnn.yaml

```diff
@@ -58,6 +58,7 @@ spec:
         - DATA.BASEDIR=/{{ .Values.maskrcnn.data_fs }}/{{ .Values.maskrcnn.data_dir }}
         - DATA.TRAIN={{ .Values.maskrcnn.data_train }}
         - DATA.VAL={{ .Values.maskrcnn.data_val }}
+        - TRAIN.GRADIENT_CLIP={{ .Values.maskrcnn.gradient_clip }}
```

Comment:
Please also update `docker/train.sh` and `ami/no_batch_train_1node_16xl_convergence.sh` as well so all training script stay consistent